packaging/vagrant/Vagrantfile

```
@@ -318,9 +318,9 @@ EOF2
     else
         user=vagrant
     fi
-    sudo -iu $user -- sh -c "ch-test clean && ch-test -b docker"
-    sudo -iu $user -- sh -c "ch-test clean && ch-test -b buildah"
-    sudo -iu $user -- sh -c "ch-test clean && ch-test -b ch-grow"
+    sudo -iu $user -- sh -c "ch-test clean && ch-test -b docker all"
+    sudo -iu $user -- sh -c "ch-test clean && ch-test -b buildah all"
+    sudo -iu $user -- sh -c "ch-test clean && ch-test -b ch-grow all"
```

I don't think we need to `clean` anymore; each phase nukes the pack/unpack dirs before running.

```suggestion
    sudo -iu $user -- sh -c "ch-test -b docker all"
    sudo -iu $user -- sh -c "ch-test -b buildah all"
    sudo -iu $user -- sh -c "ch-test -b ch-grow all"
```

---

Ah. Good point. Thanks!